README.md

```diff
@@ -32,15 +32,18 @@
 
 Program made by [lnxcz](https://github.com/lnxcz) </p>
 Improved by [DR34M-M4K3R](https://github.com/DR34M-M4K3R)
+Improved a second time by [ledges](https://github.com/ledges)
 
 <br/>
 <br/>
 
 
 # What's new?
 
-#### -When the user selects "auto-scrape proxy" option, when the proxies are invalids, the program re-scrapes the proxies, and continues the checking.
-#### -Improved the Readme :wow:
+* Massive code cleanup
+* Easily Selectable Options w/ good looking UI.
+
+`To change whether or not to scrape proxies, go to the bottom of the code and at ".generate(scrape=False") just change it to True.`
```

Comment:
don't forget to change if it will be as input

Reply:
I will go through all these changes give me 10 - 15 minutes!